docs/next/guides/upgrade-and-migration/release-notes.md

```diff
@@ -15,6 +15,36 @@ tags:
 
 [[toc]]
 
+## 12.1.0
+
+Released on 28th of June, 2022
+
+**Added**
+- Added smart positioning of comment boxes. [#4785](https://github.com/handsontable/handsontable/issues/4785)
+- Added smoother row and column moving when some row or columns are hidden. [#7613](https://github.com/handsontable/handsontable/issues/7613)
+- Added [`getNearestNotHiddenIndex()`](@/api/indexMapper.md#getnearestnothiddenindex), a new method that finds the visual index of the nearest not-hidden row or column and works even with large numbers of hidden rows or columns. The previous method, [`getFirstNotHiddenIndex()`](@/api/indexMapper.md#getfirstnothiddenindex), still works, but is marked as deprecated. [#9064](https://github.com/handsontable/handsontable/issues/9064)
```

Comment:
This is a comment to the file `api/indexMapper.md`, but I don't have an option to comment in that file, so I am doing it here:

Please proof read the entry for `getNearestNotHiddenIndex`, because some things could be improved:
- inconsistent use of `Starting point` vs `Start point`

----

In the same file, in the entry of `getFirstNotHiddenIndex`, in the following line:

```
The method is deprecated and will be removed in the nearest stable major version.
```

I think that the word "stable" is redundant.

Reply:
@warpech - ok, done: 

@budnix - can you have a quick look, please?

### `getNearestNotHiddenIndex`

> Search for the nearest not-hidden row or column.
> 
> | Param | Type | Default | Description |
> | --- | --- | --- | --- |
> | fromVisualIndex | `number` |  | The visual index of the row or column from which the search starts.<br><br> If the row or column from which the search starts is not hidden, the method simply returns the `fromVisualIndex` number. |
> | searchDirection | `number` |  | The search direction.<br><br>`1`: search from `fromVisualIndex` to the end of the dataset.<br><br> `-1`: search from `fromVisualIndex` to the beginning of the dataset (i.e., to the row or column at visual index `0`). |
> | searchAlsoOtherWayAround | `boolean` | <code>false</code> | `true`: if a search in a first direction failed, try the opposite direction.<br><br> `false`: search in one direction only. |
> 
> **Returns**: `number` | `null` - A visual index of a row or column, or `null`.

Reply:
It’s ok for me 👍